templates/cluster_resources.j2

```diff
@@ -77,6 +78,22 @@ primitive rsc_ip_{{ sid }}_ERS{{ ers_instance }} IPaddr2 \
 
 {%- endif %}
 
+{%- if monitoring_enabled %}
```

Comment:
a comment on the highelve of both resource might appreaciate

Reply:
what's not clear about this if condition?

Reply:
not the if condition but the block.. anyways if there is no comment possible on that language is ok. I mean just from external pov, is really un-readable the thing , if you don't have code it you don't know what it does :grin:

Reply:
oh come on, it's not that bad, you just have to know a little about how `crmsh` works! I didn't know how to do any of this before this PR.
What the block does is it adds two new systemd resources, one for each exporter that is collocated with an HA resource.

Reply:
I mean this whole commands are ugly, but I don't feel it mandatory. It is a nitpick but  ok. I'm alwawys for putting rendundant comment which can safe the live for not insiders but ok, I don't expect this